### DISCUSSION TITLE
FAQs ✅

### DISCUSSION THREAD

**Discussion by emlweb** [openclimatefix/community#5] [Announcements]
### Why is all this stuff open-source?
At Open Climate Fix, we're curious to see if it's possible to rapidly mitigate climate change by:

Enabling thousands of people to help solve ML problems which, if solved, might help reduce CO2 emissions
Running small(ish) pilot projects to implement the best solution in industry
Enabling thousands of practitioners to use the code in their products.

### What's the likely climate impact?
It's really, really, really hard to estimate the climate impact of forecasting! But, as a super-rough back-of-the-envelope calculation, we estimate that better solar forecasts, if rolled out globally, could reduce CO2 emissions by about a billion tonnes between now and 2035.

### How do I get involved?
Submit pull requests. Right now we are laser-focused on [our solar electricity forecasting project](https://github.com/openclimatefix/nowcasting):
[List of "good first issues"](https://github.com/search?l=&p=1&q=user%3Aopenclimatefix+label%3A%22good+first+issue%22&ref=advsearch&type=Issues&utf8=%E2%9C%93&state=open): GitHub "issues" which describe changes we'd like to make to the code
[OCF's coding style](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
The main tools we use include: PyTorch, PyTorch Lighting, xarray, pandas, pvlib
Sign up to our [newsletter](https://ocfnews.substack.com/?utm_source=substack&utm_medium=web&utm_campaign=substack_profile) and follow us on [Twitter](https://twitter.com/OpenClimateFix) and [LinkedIn](https://www.linkedin.com/company/19123036/admin/) to learn the latest about our work
Spread the word with your networks
Use our code(!) by following the guidelines below.

### How do I credit OCF if I use your code?
In order for us to understand the use of our models, and to demonstrate impact to future funders, it is invaluable for us to know who is using the code and if possible, how. We licence the code in this repository under a permissive MIT licence and if you are using the code or deriving from it, we request that you attribute the use of Open Climate Fix’s work in your product by adding the text below:
'original code by [Open Climate Fix](https://github.com/openclimatefix)'

### How do I access your data?
Our data is freely available across our repositories on GitHub. If this is your first time interacting with Open Climate Fix, here are some suggested repos to visit:

[Open-Source-Quartz-Solar-Forecast](https://github.com/openclimatefix/Open-Source-Quartz-Solar-Forecast)

[Satip](https://github.com/openclimatefix/Satip/tree/main)
Satip is a library for satellite image processing, and provides all of the functionality necessary for retrieving, and storing EUMETSAT data.

[pv-site-datamodel](https://github.com/openclimatefix/pv-site-datamodel/blob/main/README.md)
Database schema specification for PV Site data.

Our data is also available to access over on our [Hugging Face](https://huggingface.co/openclimatefix) profile.

### How do I get involved with Quartz Solar?
The aim of the project is to build an open source PV forecast that is free and easy to use. The forecast provides the expected generation in kw for 0 to 48 hours for a single PV site.
We recently presented the Quartz Solar Forecast project at FOSDEM 2024 (Free and Open source Software Developers' European Meeting), providing an introduction to Open Climate Fix's motivation for this project and its impact on aiding organisations in resource optimisation. To learn more about predictive model's functionality, visit here: [Video Recording](https://www.youtube.com/watch?v=NAZ2VeiN1N8)

**Contributions are welcome.**

Open Climate Fix also provides a commercial PV forecast, please get in touch at [quartz.support@openclimatefix.org](mailto:quartz.support@openclimatefix.org)

**Have a question that hasn't been answered above? Drop it in the comments or start a new Discussions thread in 🆘 Help**

**Comment by aryanxk02** [comment]
Hello @emlweb, I have 2 questions.

[1] Is there a discord/slack channel for openclimatefix so I can engage with maintainers and community members regarding open source development to this project?

[2] Does openclimatefix arrange virtual meetings periodically to expand interaction with interested open source contributors?

Thank you,
@aryanxk02
